server/README.md

```
@@ -30,19 +31,15 @@ docker-compose up -d
    ```bash
-   # HTTP 伺服器
+   # HTTP + WebSocket (/ws) 都由同一個 Flask/gevent server 提供
    PYTHONPATH=.. uv run python -m server.app
-
-   # WebSocket 伺服器（另一個終端）
-   PYTHONPATH=.. uv run python -m server.ws_app
    ```
 
 ## 專案結構
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)